console/src/main/webapp/manager/app/app.es6

```diff
@@ -30,6 +30,11 @@ class AppController {
     $scope.isActive = (routes) => routes.some(
       route => $location.$$path.indexOf(route) === 1
     )
+
+    Profile.get((p) => {
+      $scope.profile = p.roles.indexOf('SUPERUSER') === -1
+        ? 'index.delegated' : 'index.superuser'
```

Comment:
I understand here that a user is either a superuser or holds a delegation.
I might be wrong, but there are also regular users (with no admin rights).

Reply:
But they can't access the manager, or I'm wrong ?

Reply:
You're right, they should be filtered out by the SP. My bad for trying to follow what's going on when I'm sick...